core/src/main/java/org/apache/accumulo/core/fate/user/UserFateStore.java

```diff
@@ -105,14 +106,17 @@ public FateId create() {
         case ACCEPTED:
           return fateId;
         case UNKNOWN:
+          log.error("Fate creation returned UNKNOWN status, raising exception");
+          throw new IllegalStateException("Fate creation returned UNKNOWN status");
```

Comment:
I don't think we want to throw an exception in this case. This just means it's unknown if the mutation was successfully written or not, in which case we would want to just retry. The UNKNOWN status isn't quite equivalent to the KeeperException thrown in MFS

Reply:
Ok, I looked further at ConditionalWriter.Status.UNKNOWN and how/where it's used. I agree with your assessment and can back this out.